src/main/java/uk/gov/hmcts/reform/civil/service/SolicitorEmailValidation.java

```diff
@@ -0,0 +1,129 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.model.OrganisationPolicy;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.model.common.Element;
+import uk.gov.hmcts.reform.civil.model.genapplication.GASolicitorDetailsGAspec;
+
+import java.util.List;
```

Comment:
Any methods that are being used and meant to be used by this class only should be private

Reply:
Done